pyproject.toml

```diff
@@ -44,8 +44,11 @@ Issues = "https://github.com/secure-systems-lab/securesystemslib/issues"
 
 [project.optional-dependencies]
 crypto = ["cryptography>=37.0.0"]
+gcpkms = ["google-cloud-kms"]
 pynacl = ["pynacl>1.2.0"]
 PySPX = ["PySPX==0.5.0"]
+asn1 = ["asn1crypto"]
+pykcs11 = ["PyKCS11"]
```

Comment:
feels like this isn't the correct way to use this system. pykcs11 without asn1 doesn't make sense so why offer that as option?

Reply:
`asn1crypto` is only an immediate dependency for pubkey export. For signing we don't use it.

Reply:
But I agree, telling users to run
```
python securesystemslib[crypto, asn1, pykcs11]
```
is not a lot better, than
```
python cryptography asn1crypto pykcs11 securesystemslib
````

Reply:
Listing extra dependencies by feature, otoh, seems prone to redundancy. `cryptography`, for instance, is used for many different features.